fabric-gametest-api-v1/src/testmod/java/net/fabricmc/fabric/test/gametest/ExampleTestSuite.java

```diff
@@ -29,6 +30,9 @@ public void diamond(TestContext context) {
 		context.addInstantFinalTask(() ->
 				context.checkBlock(new BlockPos(0, 2, 0), (block) -> block == Blocks.DIAMOND_BLOCK, "Expect block to be diamond")
 		);
+
+		// This should be null for the game tests
+		assert FabricLoader.getInstance().getGameInstance() == null;
```

Comment:
Isn't `assert` useless by default? It should probably be `if (FabricLoader.getInstance().getGameInstance() != null) throw new AssertionError();`

Reply:
I might just remove this, was just a quick test tbh;

Reply:
Maybe we can just add `-ea` for vm args for test mod runs.

Reply:
Yeah, that's enabled for the regular `test` task too.